tests/run-make/jobserver-error/Makefile

```diff
@@ -0,0 +1,7 @@
+include ../../run-make-fulldeps/tools.mk
+
+# only-linux
+
+all:
+	$(RUSTC) test.rs
+	$(TMPDIR)/test $(RUSTC_ORIGINAL) $(TMPDIR) test.rs
```

Comment:
Something like `echo 'fn main()' | MAKEFLAGS="--jobserver-auth=3,3" $(RUSTC) - 3</dev/null` should work I think. I think it is clearer than writing a rust file to run effectively the same command.